### DISCUSSION TITLE
Is there a plan that supports it according to the directory?

### DISCUSSION THREAD

**Discussion by chaocwu** [pytest-dev/pytest-xdist#950] [General]
dir1
--test_dir1_1
--test_dir1_2
dir2
--test_dir2_1
--test_dir2_2
pytest -n auto --load=dir

**Comment by RonnyPfannschmidt** [comment]
its unclear what you are asking for - loadscope by packages scope?
if there is no issue about it, there is no plan

**Reply by chaocwu** [reply]
ok, thank you
